script/.env

```diff
@@ -11,4 +12,4 @@ IOS_DESTINATION_VERSION=${IOS_DESTINATION_VERSION:-"latest"}
 IOS_DESTINATION_SIMULATOR_NAME=${IOS_DESTINATION_SIMULATOR_NAME:-"iPhone 6"}
 OSX_DESTINATION_ARCH=${OSX_DESTINATION_ARCH:-""}
 INSTALL_GITHOOKS=true
-REQUIRED_XCODE_VERSION=7.3.1
+FORCE_SWIFTLINT_VERSION=true
```

Comment:
I would suggest to put this when you are calling the `script/cibuild` instead of putting it here because you do want to force the CI machine to force swiftlint update but you may not want to do that on a dev machine. The developer should be the one that explicitly decides to update the swiftlint version.